news/utils.py

```diff
@@ -54,6 +57,26 @@ def __init__(self, message, suggestion=None):
         self.suggestion = suggestion
 
 
+def get_email_block_list():
+    """Return a list of blocked email domains."""
+    cache_key = 'email_block_list'
+    block_list = email_block_list_cache.get(cache_key)
+    if block_list is None:
+        block_list = BlockedEmail.objects.values_list('email_domain', flat=True)
```

Comment:
micro-optimization NIT: I would go ahead and convert this to an actual list-- no point in pickling/unpickling the queryset object itself.  Since you're using LocMemCache the performance impact will be negligible, so not a blocker, but it will help if you ever switch to memcache or redis because you'll be sending less data over the wire.